backend/processing/content_iterator/task.py

```diff
@@ -0,0 +1,33 @@
+"""
+TODO: add docstring here.
+
+"""
+
+
+def lambda_handler(event, _context):  # pylint:disable=unused-argument
```

Comment:
You don't need this pylint annotation here.

Reply:
Done in 599f367570528d25a8e23b358e86182fe62eca96